### PR TITLE
Rift Race and Wooden Buttons Quest

### PR DESCRIPTION
**Pull request by MTOnline69**
Dependency for https://github.com/hannibal002/SkyHanni/pull/2616

### CODE REVIEW

**Comment by hannibal002** [comment]
please move the two new json files into a newly created folder "rift"

File: constants/RiftWoodenButtons.json [hunk 1]
```diff
@@ -0,0 +1,140 @@
+{
+  "houses": {
+    "Mirrorverse House": [
```
Comment (hannibal002) [inline constants/RiftWoodenButtons.json:3]: if all names have the suffix house, you can remove it and include it in code

**Review by hannibal002 — CHANGES_REQUESTED** [review]

**Comment by MTOnline69** [comment]
> please move the two new json files into a newly created folder "rift"

i assume effigies and mirrorverse ones too?

**Comment by hannibal002** [comment]
> i assume effigies and mirrorverse ones too?

No. Only newly added files. Changing the path from old data will immediately break every older sh version

**Pull request merged**